### PR TITLE
PLATML-5948 - Incorporate sandbox-related logic into Scala sample 

### PR DESCRIPTION
**Pull request by NemanjaL94**

### CODE REVIEW

File: recipes/scala/pom.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
 		<dependency>
 			<groupId>com.adobe.platform.ml</groupId>
 			<artifactId>authoring-sdk_2.11</artifactId>
-			<version>0.21.5</version>
+			<version>0.21.6-SNAPSHOT</version>
```
Comment (NemanjaL94) [inline recipes/scala/pom.xml:53]: Snapshot until release

**Comment by tumulurik** [comment]
LGTM, but waiting to approve until we have clarity on the platform SDK version can run on HDI.

**Comment by ichasepucks** [comment]
This PR should likely be closed since this is not destined for public consumption for quite some time.  Internal PR https://git.corp.adobe.com/ml/experience-platform-dsw-reference/pull/2 incorporates these changes.

**Comment by NemanjaL94** [comment]
@ichasepucks Got it, I'll close this PR

**Pull request closed without merging**